addons/udes_stock/README.md

```diff
@@ -95,62 +95,79 @@ The type of stock.picking can is defined by this type. It can represent a goods
 
 A lot of custom UDES functionality is specfied at the picking type level. This is where the stock storage format is specified, so the system knows how to store stock (i.e. as just products, in packages or pallets).
 
-| Field Name                | Type    | Description                                       |
-| ------------------------- | ------- | ------------------------------------------------- |
-| id                        | int     | |
-| code                      | string  | |
-| count_picking_ready       | int     | |
-| default_location_dest_id  | int     | |
-| default_location_src_id   | int     | |
-| display_name              | string  | |
-| name                      | string  | |
-| sequence                  | int     | Used for ordering picking types in a display. |
-| u_allow_swapping_packages | boolean | During a specified pick, this field determines whether we can we swap one package for another if they contain exactly the same. |
-| u_skip_allowed            | boolean | Is the user allowed to skip to the next item to pick? |
-| u_split_on_drop_off       | boolean | |
-| u_suggest_qty             | boolean | Do we display the suggested quantity, or get the user to enter it without any vision of what is expected. When we suggest it, the risk is that users will be automatically confirming it without a thorough check. |
-| u_over_receive            | boolean | Is the system able to receive more than is expected. |
-| u_target_storage_format   | string  | This defines how the stock is stored at the end of the stock.picking. |
-| u_user_scans              | string  | This defines what the user will scan. |
-| u_validate_real_time      | boolean | Do we validate move lines in real time |
+
+
+// TODO: REMOVE VVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVV
```

Comment:
Remove them :)

Reply:
When I don't need that anymore.
BTW, don't worry, as there a TODO item in the main thread to avoid forgetting that.